docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -259,59 +272,59 @@ Account Global Admin has Namespace Admin permissions on Namespaces.
+| GetRegion                             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRegions                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatus                      | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatuses                    |                    |           |               | ✔            | ✔             |
+| GetRequestStatusesForNamespace        | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatusesForUser             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRoles                              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRolesByPermissions                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccount                     | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccounts                    | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetStripeInvoice                      |                    |           | ✔             |              | ✔             |
+| GetUser                               | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetUsers                              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetUsersWithAccountRoles              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| InviteUsers                           |                    |           |               | ✔            | ✔             |
+| ListCreditLedgerEntries               |                    |           | ✔             |              | ✔             |
+| ListGrants                            |                    |           | ✔             |              | ✔             |
+| ListMetronomeInvoices                 |                    |           | ✔             |              | ✔             |
+| ListMetronomeInvoicesForNamespace     |                    |           | ✔             |              | ✔             |
+| ListNamespaces                        | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| ListPromotionGrantBalances            |                    |           | ✔             |              | ✔             |
+| ResendUserInvite                      |                    |           |               | ✔            | ✔             |
+| SetAccountSettings                    |                    |           |               | ✔            | ✔             |
+| SyncCurrentUserInvite                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
```

Comment:
We don't really use this api for anything I don't think it's even part of the public API?

Also just thinking this through--for the API table here should we only be listing the publicly available cloud ops apis?

https://github.com/temporalio/api-cloud

Reply:
https://github.com/temporalio/api-cloud/blob/main/temporal/api/cloud/cloudservice/v1/service.proto